internal/helm/release/manager.go

```diff
@@ -248,7 +248,7 @@ func reconcileRelease(_ context.Context, kubeClient kube.Interface, expectedMani
 		}
 
 		helper := resource.NewHelper(expected.Client, expected.Mapping)
-		existing, err := helper.Get(expected.Namespace, expected.Name, expected.Export)
+		existing, err := helper.Get(expected.Namespace, expected.Name)
```

Comment:
Does `helper.Get()` behavior change without `expected.Export`?

Reply:
Previously, based on `expected.Export` `export` parameter would be added in Get request. Controller-runtime has removed that and modified the function signature.